Replaced st.experimental_get_query_params with st.query_params

Suvid-Singhal opened the pull request:
Replaced st.experimental_get_query_params with st.query_params as st.experimental_get_query_params was deprecated. This ensures that the API demo app still works.
Fixes #6

Suvid-Singhal commented:
Hi @anth-volk,
Could you please have a look at it? 😄

anth-volk reviewed (changes requested):
Thanks for your contribution here, @Suvid-Singhal. Unfortunately, I ran these changes, and they actually crash. Per the documentation at https://docs.streamlit.io/develop/api-reference/caching-and-state/st.query_params, fetching query params in the new version of Streamlit requires a different structure, which these changes do not include. Would you mind updating based on that?

Suvid-Singhal commented:
Hi @anth-volk,
I'll be able to look into this issue after 2 days as my exams are going on.
I'll surely try to fix it :)

Suvid-Singhal commented:
Hi @anth-volk,
Sorry for such a late response. I was caught up with exams.
I've fixed the errors now and implemented it properly this time.
It now works as intended.
Could you please have a look at it?

anth-volk commented:
Sure, thanks @Suvid-Singhal, will look into either Monday or Tuesday

anth-volk reviewed (approved):
Thanks for this, @Suvid-Singhal! This looks like it's working now. I'll go ahead and push the changes.
